src/main/java/science/icebreaker/config/MyAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,23 @@
+package science.icebreaker.config;
+
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+/**
+ * This implementation of {@link AuthenticationEntryPoint} can be used to stop the framework's authentication
+ * process.
```

Comment:
How can we stop authentication process? Do we need to make changes to the SecurityConfig? Provide more documentation